packages/react/test/specs/components/Accordion/AccordionTitle-test.tsx

```diff
@@ -0,0 +1,143 @@
+import * as React from 'react'
+import * as _ from 'lodash'
+
+import AccordionTitle from 'src/components/Accordion/AccordionTitle'
+import { isConformant, handlesAccessibility, getRenderedAttribute } from 'test/specs/commonTests'
+import { mountWithProviderAndGetComponent } from 'test/utils'
+
+describe('AccordionTitle', () => {
+  isConformant(AccordionTitle, {
+    eventTargets: {
+      onClick: `.${AccordionTitle.slotClassNames.button}`,
```

Comment:
Why is `onClick` a string?

Reply:
it says that the handler will go to that slot and not on the component. otherwise the isConformant will fail and tell that onClick is not applied to ElementType.

Reply:
Thanks for the clarification, missed that this is used as a config for the conformat test. Will resolve this one :)